### DISCUSSION TITLE
Help wanted (at snow)

### DISCUSSION THREAD

**Discussion by baitmooth** [Arcticons-Team/Arcticons#2245] [Q&A] [answered]
Hej there. I'm the maintainer of [Snow](https://github.com/baitmooth/snow). The project was dormant for a while because of personal reasons. Now that i am back at working on it, and got some stuff done already, I find myself in the position were I could need some help.
At the moment this breaks down to mostly programming:
I saw that Arcticons uses some nice Python scripts to do work on the svg and xml files. I'd love to adopt those for snow but I fail in doing so on my own. So if one might help me out on this … <3

And not to mention i love how Arcticons is looking growing … 

best wishes 
baitmooth

**Comment by Kaiserdragon2** [comment]
Hi i can help you with getting the scripts portet to snow. Are there some things in particular that you need or just the handling of the conversion of svg to png/xml sorting the appfilter, categories and drawable.xml creation.

**Reply by baitmooth** [reply]
Hej, that would be awesome.
Nothing special I need. I only have white icons, so the conversion to black is not necessary. (Starting with having WHITE_DIR/BLACK_DIR, only need ICON_PATH.)

 Besides that i guess it's like here at Arcticons.

**Reply by Kaiserdragon2** [reply, marked as answer]
https://github.com/baitmooth/snow/issues/27

**Reply by Donnnno** [reply]
@baitmooth great to see you working on Snow again! I hope you're doing well now. :)
Great to see that you're already getting help with the code stuff. I'm still lacking the right knowledge when it comes to scripting. But @Kaiserdragon2 has evolved Arcticons time and time again :)) 

( @Kaiserdragon2 you're the best! )

**Reply by baitmooth** [reply]
THX @Donnnno.
had some rough times … not that it's less rough atm, but I guess I can deal with it better now ;)

I saw all your great work from the user perspective over the past years. ya all are amazing. And while there are great people like @Kaiserdragon2 it's not really necessary to to understand code deeper than two levels of indention.
